doc/chapter-configuration.asciidoc

```diff
@@ -0,0 +1,211 @@
+Several aspects of Newsboat can be configured via a _config_ file,
+which is stored next to the _urls_ file.
+A configuration line looks like this in general
+
+`<config-command> <arg1> ...`
+
+The configuration file can contain comments, which start with the `+#+` character and go as
+far as the end of line.
+
+****
+*User contrib* Newsboat also comes with user contributed content like scripts and color
+themes. The user contributed content can be found in
+`/usr/share/doc/newsboat/contrib/`. End users are encouraged to take a look as
+they may find something useful.
+****
+
+=== Example
+
+An example configuration looks like this
+
+----
+# This is a comment
+max-items 100 # Inline comments are possible, too
+unbind-key R # Single argument config
+bind-key ^R reload-all # Multi argument config
```

Comment:
I agree that the comments do not explain much. I guess I just strived for a "minimal" version of a config file? I'm happy to remove the comments but I think for an Example config file it would actually be nice to show some more config options (some key bindings, formatting, colors etc.) What do you think?

Reply:
I agree. However, this PR is large enough already. Let's change this example config back to how it was originally, and then you can submit a separate PR with an expanded config. Does that sound acceptable?

Reply:
Sounds good to me :+1: Done